packages/lego-bricks/src/components/Layout/Page/Page.tsx

```diff
@@ -63,7 +63,7 @@ const Page = ({
         className={title ? styles.title : undefined}
       >
         {sidebar?.side === 'left' && <SidebarTrigger />}
-        {skeleton ? (
+        {skeleton && !title ? (
```

Comment:
no need to show skeleton if we have a title

Reply:
Eeeeeh, idk. I think we might get a lot of f.ex. "Editing: undefined" while loading.

Reply:
Yes that's why I also edited how the title on the event administrate pages are set.

However, I do believe it's a bad use of the `Page` component though if you manage to get "Editing: undefined". Would you agree to a hacky, but viable, solution like

```suggestion
        {skeleton && (!title || title.includes("undefined"))  ? (
```

Reply:
It's just annoying that the title starts "loading" when switching tabs